examples/hello-world/ml-to-fl/tf/tf_client_api_job.py

```diff
@@ -51,7 +52,8 @@ def main():
     executor = ScriptRunner(
         script=script,
         launch_external_process=launch_process,
-        framework=FrameworkType.TENSORFLOW,
+        server_client_params_exchange_format=ExchangeFormat.NUMPY,
+        client_script_params_exchange_format=ExchangeFormat.KERAS_NUMPY,
```

Comment:
Do we need 2 parameters? server/client should either use the same format or convert to the final format (script). Otherwise, we may have 3 formats involved.

Reply:
Before we have some auto/default mechanism, I think we do need two formats: 
- what the script expects the input to be
- what the server expects the response to be

Since the client (CJ) only plays as a middleman, we probably should rename these two formats:
server_client_params_exchange_format => server_expected_format
client_script_params_exchange_format => script_expected_format

Reply:
Renamed as suggested.
Right now we already have the following combinations:

SJ <-NUMPY-> CJ <-PYTORCH-> SCRIPT
SJ <-PYTORCH-> CJ <-PYTORCH-> SCRIPT

So we need a way to express these as well.